stories/core_components/BulletChart.tsx

```diff
@@ -0,0 +1,288 @@
+// (C) 2007-2020 GoodData Corporation
+import * as React from "react";
+import { storiesOf } from "@storybook/react";
+import { screenshotWrap } from "@gooddata/test-storybook";
+
+import { BulletChart } from "../../src";
+import { onErrorHandler } from "../mocks";
+import { CUSTOM_COLORS } from "../data/colors";
+import {
+    ATTRIBUTE_1,
+    ATTRIBUTE_2,
+    MEASURE_1,
+    MEASURE_1_WITH_ALIAS,
+    MEASURE_2,
+    ATTRIBUTE_1_SORT_ITEM,
+    MEASURE_2_SORT_ITEM,
+} from "../data/componentProps";
+import { GERMAN_SEPARATORS } from "../data/numberFormat";
+import { CUSTOM_COLOR_PALETTE_CONFIG } from "../data/configProps";
+import { Execution } from "@gooddata/typings";
+import { RGBType } from "@gooddata/gooddata-js";
+
+const wrapperStyle = { width: 800, height: 400 };
+
+storiesOf("Core components/BulletChart", module)
```

Comment:
there are several features of the bar chart which should be working on the bullet chart too. For instance
 - attribute grouping
 - axis hiding, rotating labels, min/max values
 - custom colors
 - sorting
So here are those that should be supported at this stage of development of the bullet chart
Some will be added later (Arithmetic measures, over time comparison)
Some won't (Stacking)